rust/exchange_rate/README.md

```diff
@@ -0,0 +1,10 @@
+## Build the canister wasm file
+`cd rust/exchange_rate`
+`wasm-pack build --out-dir ./wasm/ --out-name exchange_rate.wasm --release`
+## Deploy the canister
+`dfx deploy`
+
+# Mainnet exchange_rate canister Id
+mfwph-oqaaa-aaaam-qafsq-cai
+
+Check the status of the canister: `dfx canister --network=ic status mfwph-oqaaa-aaaam-qafsq-cai`
```

Comment:
Could you please add some description what this example is all about?

Reply:
Yes, added purpose of the dapp, how to use, and canister behavior explanations.